plugins/core/Standard/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ set( submod_plugins
 		${CMAKE_CURRENT_SOURCE_DIR}/qMasonry
 		${CMAKE_CURRENT_SOURCE_DIR}/qMPlane
 		${CMAKE_CURRENT_SOURCE_DIR}/qJSonRPCPlugin
+		${CMAKE_CURRENT_SOURCE_DIR}/qUltrabeam
```

Comment:
Is qUltrabeam a part of the qDracoIO plugin ? 🤔

Reply:
Good point, this one shouldn't even be here 👍